src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/sts_stack.py

```diff
@@ -10,9 +10,17 @@ def __init__(self, scope: core.Construct, id: str, common_stack: CommonStack, **
 
         self._supported_in_region = self.is_service_supported_in_region()
 
+        assumed_by_principal = common_stack.circleci_execution_role
+        if common_stack.github_action_oidc is not None:
```

Comment:
can `common_stack` be `None` here?

Reply:
No, the `StsStack` is created under `app.py`, the `common_stack` is created and passed in.